api_tests/registrations/views/test_registration_list.py

```diff
@@ -722,6 +722,18 @@ def test_admin_can_create_registration(
         }
         assert data['public'] is False
 
+    @mock.patch('framework.celery_tasks.handlers.enqueue_task')
+    def test_draft_registration_retains_title_after_reg(
+            self, mock_enqueue, app, user, payload, url_registrations, draft_registration):
+        draft_registration.title = 'draft reg title'
+        draft_registration.save()
```

Comment:
Could you either add checks in this test for the other fields touched by `copy_editable_fields`, or a unit test specifically for that function?
Other fields are `description`, `category`, `node_license`, (non-creator-`contributor.`)`unclaimed_records`, `tags`, `subjects`, and `affiliated_institutions`